Client/mods/deathmatch/logic/CClientRadarArea.cpp

```diff
@@ -117,3 +117,14 @@ void CClientRadarArea::RelateDimension(unsigned short usDimension)
 {
     m_bStreamedIn = (usDimension == m_usDimension);
 }
+
+void CClientRadarArea::SetInterior(unsigned char ucInterior)
+{
+    m_ucInterior = ucInterior;
```

Comment:
Check if you have to update the interior here: `if (m_ucInterior != ucInterior)`